tests/test_client.py

```diff
@@ -1201,6 +1201,21 @@ def test_exchange_success(self):
         self.assertEqual('dummy_revoke_uri', credentials.revoke_uri)
         self.assertEqual(set(['foo']), credentials.scopes)
 
+    def test_exchange_success_binary_code(self):
+        payload = (b'{'
+                   b'  "access_token":"SlAV32hkKG",'
```

Comment:
Please don't repeat the literal values of your test data. Please instead create local constants for your test data values and then use those constants both where you supply the test data to the system under test and where you confirm the correctness of the system under test.

I recognize that this is asking you not to follow the pattern of the other test methods nearby.